icons/shield-lock.json

```
@@ -0,0 +1,24 @@
+{
+  "$schema": "../icon.schema.json",
+  "contributors": [
+    "caisere"
+  ],
+  "tags": [
+    "safeguard",
+    "securitysystem",
```

Spaces are allowed in tags:
```suggestion
    "security system",
```